apps/tup-ui/src/pages/Projects/Projects.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ProjectsLayout, RequireAuth } from '@tacc/tup-components';
+
+const Layout: React.FC = () => {
+  return (
+    <RequireAuth>
+      <div>
+        <ProjectsLayout />
+      </div>
```

Comment:
I recognize this is probably copied from `Dashboard.tsx`. Extra wrappers (i.e. superfluous `<div>`) often makes _modern_ CSS more cumbersome than it needs to be. Please leave such `<div>`s out until they prove necessary.

<sup>The Dashboard `<div>` might actually be necessary… (note to self) unless I can get `<body>` classes toggled per route.</sup>